_docs/geyser/getting-started-with-api.md

```diff
@@ -1,29 +1,47 @@
 ---
-title: Using Geyser or Floodgate as a dependency
+title: Getting Started with Geyser's and Floodgate's API
 ---
 
 To start, add the Open Collaboration repository to your project:
```

Comment:
Maybe add a note that this `main` repo includes both release and snapshot versions